dependencies/direct/py.in

```diff
@@ -9,3 +9,4 @@ pytest
 pytest-cov
 pytest-mock
 pytest-xdist
+PyGithub
```

Comment:
This is a wrong place for the dep. It must be in the core packaging metadata, not just a testing-only thing.